schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -650,6 +650,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Sql.json#/resourceDefinitions/servers_jobAgents_targetGroups" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces" },
                   { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces_AuthorizationRules" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces_NetworkRuleSets" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2017-04-01/Microsoft.ServiceBus.json#/resourceDefinitions/namespaces_NetworkRuleSets" }, [](start = 18, length = 143)

same comment. can you please add to the new root schema 2019-04-01/deploymentTemplate.json too?

Reply:
Resolved : Resolved. Added in  2019-04-01/deploymentTemplate.json